modules/ROOT/pages/procedures.adoc

```diff
@@ -249,7 +249,7 @@ For more information, see the link:{neo4j-docs-base-uri}/cdc/{page-version}/[CDC
 | *Mode* 3+| READ
 |===
 
-[role=label--new-5.17 label--beta]
+[role=label--beta]
```

Comment:
Do we need this label here? The CDC feature is GA starting from 5.23.

Reply:
If it is GA, no.